### ISSUE TITLE
launch-validator-tools-with-binaries.sh blows up if you supply mnemonics more than once

### ISSUE THREAD

**Issue by ggutoski** [axelarnetwork/axelarate-community#278]
## Describe the bug

You can only run `./join/launch-validator-tools-with-binaries.sh --proxy-mnemonic broadcaster_mnemonic.txt --tofnd-mnemonic tofnd_mnemonic.txt` exactly once after resetting the chain.

Killing and attempting to restart after that first time, results in errors and the companion processes will not start. Starting WITHOUT the flags results in the script creating  a new broadcaster address and mnemonic (which is not what you're trying to do).

## Expected behavior

Reasonable behaviour if mnemonics are supplied multiple times.  Examples:
1. Ignore mnemonics and print a message explaining why
2. Ask user if he/she wishes to overwrite existing mnemonics and re-derive accounts. (This probably opens a can of worms.)

## Workaround
[Validators on discord](https://discord.com/channels/770814806105128977/867457262620770334/922546846377857055) painstakingly deduced the following workaround that recovers the desired state in the current axelar-core:

```
kill -9 $(pgrep -f "axelard vald-start") && kill -9 $(pgrep -f "tofnd")`
rm -rf .axelar_testnet/.tofnd/kvstore
rm .axelar_testnet/.vald/keyring-test/*

# This might not have been required
# [Gus: Seems like it is required.  Link below.]
# rm .axelar_testnet/broadcaster.bech

./join/launch-validator-tools-with-binaries.sh --proxy-mnemonic broadcaster_mnemonic.txt --tofnd-mnemonic tofnd_mnemonic.txt
```

Another discord validator [confirms](https://discord.com/channels/770814806105128977/867457262620770334/921984042344263690) that `rm .axelar_testnet/broadcaster.bech` is in fact needed.

**Comment by ggutoski** [comment]
no longer relevant after #289
